hosting/src/components/Tiptap/BubbleMenu.tsx

```diff
@@ -1,16 +1,329 @@
+            className={baseStyleButton}
+            id="menu-button"
+            aria-expanded="true"
+            aria-haspopup="true"
+            onClick={toggleDropdownFormat}
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="currentColor"
+              strokeWidth="2.5"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              className="w-4 h-4"
+            >
+              <path d="M13 4v16"></path>
+              <path d="M17 4v16"></path>
+              <path d="M19 4H9.5a4.5 4.5 0 0 0 0 9H13"></path>
+            </svg>
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="currentColor"
+              strokeWidth="2.5"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              className="w-2 h-2"
+            >
+              <path d="m6 9 6 6 6-6"></path>
+            </svg>
+          </button>
+
+          <div className="bg-neutral-200 dark:bg-neutral-800 h-full min-h-[1.5rem] w-[1px] mx-1 first:ml-0 last:mr-0"></div>
+
+          <span>
+            <button
+              className={editor.isActive("bold") ? `${baseStyleButton} is-active` : baseStyleButton}
+              onClick={() => editor.chain().focus().toggleBold().run()}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+                strokeWidth="2.5"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                className="w-4 h-4"
+              >
+                <path d="M14 12a4 4 0 0 0 0-8H6v8"></path>
+                <path d="M15 20a4 4 0 0 0 0-8H6v8Z"></path>
+              </svg>
+            </button>
+          </span>
+
+          <span>
+            <button
+              className={editor.isActive("italic") ? `${baseStyleButton} is-active` : baseStyleButton}
+              onClick={() => editor.chain().focus().toggleItalic().run()}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+                strokeWidth="2.5"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                className="w-4 h-4"
+              >
+                <line x1="19" x2="10" y1="4" y2="4"></line>
+                <line x1="14" x2="5" y1="20" y2="20"></line>
+                <line x1="15" x2="9" y1="4" y2="20"></line>
+              </svg>
```

Comment:
Honest question: is there a benefit to having SVG inline instead of moving it to an assets folder when there aren't any dynamic attributes?

Reply:
To be honest, there is no benefit other than just speeding me up when copying and pasting SVG content and then inserting it directly into the code and this is not best practice for loading SVG.
It would be a hassle if this couldn't be a reusable file. I'll update this

Reply:
Thanks. Because I've seen it in many places that there seems to be some preference of inlining it. I'm personally preferring assets file option because the file name helps telling what the icon is.

Reply:
Ticket created https://github.com/oddbit/tanam/issues/395